chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -456,6 +456,15 @@ if [ "$MODE" == "setup" -o "$MODE" == "start" ]; then
     # Get our hostname
     HOSTNAME=$(hostname -f)
 
+    # stop ntpd before we run ntpdate, and start it again afterwards.
+    ntp=ntp
+    service ntp status > /dev/null || ntp=ntpd
```

Comment:
There's also some `<% if @target_platform_version.to_f < 12.0 %>` magic in the template, so why not use this to define the name of the service?